analytics/src/analytics/metrics/percent_complete.py

```diff
@@ -50,8 +41,8 @@ def calculate(self) -> pd.DataFrame:
         5. Divide closed count by total count to get percent complete
         """
         # get total and closed counts per deliverable
-        df_total = self._get_count_by_deliverable(status="all", unit=self.unit)
-        df_closed = self._get_count_by_deliverable(status="closed", unit=self.unit)
+        df_total = self._get_count_by_deliverable(status="all")
```

Comment:
nice to see this cleaned up with the new home for `unit` on `self`

Reply:
Yeah that was a happy side-effect of the switch to using the enum everywhere!